migrations/20180918144616_add_alias_to_sections.js

```diff
@@ -0,0 +1,17 @@
+exports.up = async function(knex) {
+  await knex.schema.table("Sections", table => table.string("alias"));
+  await knex.raw(`DROP VIEW "SectionsView"`);
+  await knex.raw(`

```

Comment:
Can we return the Promise here, not 100% sure it matters but I think things can sometimes go wrong when we don't return a promise from a knex migration.